core/frontend/src/core-frontend.ts

```diff
@@ -218,3 +218,7 @@ export * from "./RealityDataSource";
  * APIs for working with user preferences in an iModelApp.
  * See [the learning articles]($docs/learning/frontend/preferences.md).
  */
+/**
+ * @docs-group-description RealityDataSource
```

Comment:
We don't need a doc group just for this one file. Please remove RealityDataSource doc group and put the following just below the copyright header in RealityDataSource.ts:
```
/** @packageDocumentation
 * @module Tiles
 */
```
In the future we may want to consider breaking the "Tiles" doc group into smaller groups, one of which might contain reality-data-related APIs.

Reply:
Done, remove RealityDataSource doc group and add @packageDocumentation